solution/WellFired.Guacamole.Unity.Editor/NativeControls/Views/TextEntryViewRenderer.cs

```diff
@@ -63,8 +58,10 @@ public override void Render(UIRect renderRect)
 			else
 				SetupWithNonePlaceholderStyle(entry, Style);
 
-			var controlId = GUIUtility.GetControlID(FocusType.Keyboard) - 1;
-			var isFocused = controlId == textEditorControlId;
+			var isFocused = false;

```

Comment:
Since you've combined the functionality here of isFocused and does activeTextEditor exit, I think you need to rename this variable, I had to look through this code again to check how you were doing null ref protection ;)